MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDelegate.java

```diff
@@ -95,7 +95,7 @@ public float getElevation() {
      * @see View#setElevation(float)
      */
     public void setElevation(float elevation) {
-        boolean needsWrap = elevation > mElevation;
+        boolean needsWrap = elevation != mElevation;
```

Comment:
I'm also having a bit of a hard time understanding this change.

Some context: In _theory_, we only need to re-wrap when elevation _increases_. The reason is that it's only in that case that we need more padding, hence we'd need to reset the drawable for the view to update it's padding. If _smaller_, we could reuse the same padding, albeit drawing in a smaller area.

So, what is the behavioral change with and without this change?